blynk-app/main.cpp

```diff
@@ -164,29 +164,35 @@ static void PopulateSessionDropDown() {
 BLYNK_CONNECTED() {
    int i;
 
-   // Blynk.syncVirtual(SOUNDCARD); //sync Soundcard selection
-   Blynk.syncVirtual(SAMPLE_RATE); //sync sample rate on connection
+   // Sync buttons that control alsa first, don't need jack started for this
    Blynk.syncVirtual(INPUT_LEVEL); //sync input level on connection
    Blynk.syncVirtual(OUTPUT_LEVEL); //sync output level on connection
    Blynk.syncVirtual(INPUT_SELECT); //sync Input selection
+   Blynk.syncVirtual(MIC_GAIN);  //sync Mic Gain
+   
+ 
+   Blynk.syncVirtual(SAMPLE_RATE); //sync sample rate on connection
+   
+      // set connection states to off
+   for (i=0; i<TOTAL_SLOTS; i++) {
+      Blynk.virtualWrite(connectButton[i], LOW);
+ //     Blynk.virtualWrite(gainSlider[i], 0);
+   }
 
+   //Sync Slot settings
    for (i=0; i<TOTAL_SLOTS; i++) {
-      Blynk.syncVirtual(roleButton[i]); //sync Client 1 IP
+      Blynk.syncVirtual(roleButton[i]); //sync Role
+	  Blynk.syncVirtual(latencySlider[i]); //sync Latency
+	  Blynk.syncVirtual(gainSlider[i]); //sync Gain
+	  	  
    }
 
    // initialize menu items
    PopulateSessionDropDown();
-   Blynk.syncVirtual(SESSION_DROP_DOWN); 
+   //Blynk.syncVirtual(SESSION_DROP_DOWN);  --I don't think we want to sync the session dropdown, we want to sync to the current settings in the app, I  think if the user wants to revert to session settings they would re-select
```

Comment:
Move this comment to it's own line above this one.  Try to keep line lengths to <= 80 characters.